doc/dev/developer_guide/issue-tracker.rst

```diff
@@ -1,3 +1,5 @@
+.. _issue-tracker:
```

Comment:
looks like this ref didn't work for some reason (is there a dash-space conversion?):

```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/developer_guide/basic-workflow.rst:198: WARNING: Unknown target name: "issue tracker".
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/developer_guide/basic-workflow.rst:303: WARNING: Unknown target name: "issue tracker".
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/internals.rst:18: WARNING: duplicate label mailing-list, other instance in /home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/developer_guide/essentials.rst```

Reply:
no, my bad. fixed and repushed.